### PR TITLE
docs: fixed some typo and grammatical errors in README.MD

### PR DESCRIPTION
**Pull request by Creatoon**
**What kind of change does this PR introduce?**

**Did you add tests for your changes?**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

**Comment by Creatoon** [comment]
@okonet @andersk @sapegin I have fixed some typo errors and grammatical errors in README.MD file so that other contributors can understand it. and this is my first contribution to this organization . and very excited to make future contributions

**Comment by sapegin** [comment]
Congrats on the first pull request! However, mentioning random folks, who have no relation the project, isn't going to help you.

**Comment by Creatoon** [comment]
@sapegin OK OK sir,  I didn't knew that,  I will not to this again,  thank you

**Review by evenstensberg — APPROVED** [review]

**Comment by evenstensberg** [comment]
Congrats!

**Review by evilebottnawi — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -48,11 +48,11 @@ Otherwise `npm install --save-dev webpack-cli` or `yarn add webpack-cli --dev` w
 
 ## Supported arguments and commands
 
-Get to know what are the available commands and arguments [here](./packages/webpack-cli/README.md).
+Get to know about: what are the available commands and arguments [here](./packages/webpack-cli/README.md).
```
Comment (anshumanv) [inline README.md:51]:
```suggestion
All the available commands and arguments are [here](./packages/webpack-cli/README.md).
```

File: README.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Thus, webpack CLI provides different commands for many common tasks.
 
 ### Utilities
 
-The project also has several utility packages which are used by other commands
+The project also have several utility packages which are used by other commands
```
Comment (anshumanv) [inline README.md:77]: prev is correct

Comment (snitin315) [reply README.md:77]: Yes, `has` is correct.
Screenshot at 2020-08-28 18-26-33

File: README.md (hunk 1 above)
Comment (Creatoon) [reply README.md:51]: OK sir,  let me fix this

**Comment by Creatoon** [comment]
@snitin315  sir , is there any mistake happened as i pushed the changes to the same branch by force (using -f flag) , i don't  know how this branch got closed ?

**Pull request closed without merging**